src/logic/safe/store/actions/createTransaction.ts

```diff
@@ -160,45 +188,25 @@ export const createTransaction =
           txHash = hash
           dispatch(closeSnackbarAction({ key: beforeExecutionKey }))
 
-          await saveTxToHistory({ ...txArgs, txHash, origin })
+          try {
+            await saveTxToHistory({ ...txArgs, origin })
+          } catch (err) {
+            showError(err)
+            return
```

Comment:
I don't know if we should return here. The transaction can technically succeed even w/o notifying the backend. WDYT, @dasanra?

Reply:
this is true only for execution transaction (in this case only for 1 owner safe)

Reply:
I tested it with regular send funds and noticed that the tx succeeded according to MetaMask even though this request failed.

Reply:
the backend stores transaction information (data, recipient, value, etc), and on the contract level we approve hashed transaction data from the backend

for the safe with threshold 2, if the first owner signs tx off-chain and request to the server fail, the second owner won't see the transaction

if a transaction is executed on-chain, it will be indexed by the backend because you can get all field values (data, recipient, etc) from the blockchain itself

Reply:
OK, so `if (!isExecution) { return; }` should be correct then. Thanks for the explanation!

I'll test both cases on Monday to make sure.